lib/tasks/packager.rake

```diff
@@ -63,7 +63,8 @@ namespace :packager do
     # Persist configuration
     Setting.sys_api_enabled = 1
     Setting.sys_api_key = ENV.fetch("SYS_API_KEY", nil)
-    Setting.host_name = ENV.fetch("SERVER_HOSTNAME", Setting.host_name) if Setting.host_name_writable?
+    host_name = ENV.fetch("SERVER_HOSTNAME", Setting.host_name)
+    shell_setup(["config:set", "OPENPROJECT_HOST__NAME=#{host_name}"])
```

Comment:
Since we now share ENV + settings, this is the better way to define the host name in a way it is no longer writable